modules/sdk/gradle-plugins-jsdoc/src/main/java/com/liferay/gradle/plugins/jsdoc/JSDocTask.java

```diff
@@ -71,9 +77,11 @@ public File getReadmeFile() {
 		return GradleUtil.toFile(getProject(), _readmeFile);
 	}
 
-	@InputDirectory
-	public File getSourcesDir() {
-		return GradleUtil.toFile(getProject(), _sourcesDir);
+	@InputFiles
+	public FileCollection getSourcesDirs() {
```

Comment:
Can you please call the property `sourceDirs` instead of `sourcesDirs`? Brian already approved `sourceDirs` in other plugins, so I think it's the way to go 😊